lib/v2/datasource/remote/api/signup_repository.dart

```diff
@@ -69,4 +75,37 @@ class SignupRepository extends NetworkRepository {
             }))
         .catchError((error) => mapHttpError(error));
   }
+
+  Future<Result> createAccount({String? accountName, String? inviteSecret, String? displayName}) async {
```

Comment:
Can these parameters be null?

Reply:
No they can't be null, although displayname can be "". Not null though.

Reply:
Then:
`createAccount({String accountName, String inviteSecret, String displayName = ''})`

Reply:
Addressed